testsuite/integration/microprofile/pom.xml

```diff
@@ -661,6 +782,12 @@
                                             org.wildfly.arquillian:wildfly-arquillian-container-managed
                                         </classpathDependencyExclude>
                                     </classpathDependencyExcludes>
+                                    <excludes>
+                                        <exclude>org/wildfly/test/integration/microprofile/jwt/ejb/JWTEJBTestCase.java</exclude>
+                                        <exclude>org/wildfly/test/integration/microprofile/config/smallrye/converter/MicroProfileConfigConvertersTestCase</exclude>
```

Comment:
maybe, we should also add the comments that explains why these tests are excluded (as they require the creation of a JBoss Module)

Reply:
I investigated the tests and module.path was not set to the right location, I fixed that. I removed the exclusion for bootable.jar, 1 test still fails MicroProfileConfigConvertersTestCase with ts.bootable.ee9. This needs to be investigated. @bstansberry , I tried with the ee9 profile and MicroProfileConfigConvertersTestCase as well MicroProfileConfigTestCase are failing.

Reply:
Thanks. The nightly for WildFly Preview shows those 2 failures.

https://ci.wildfly.org/viewLog.html?buildId=234874&tab=buildResultsDiv&buildTypeId=WF_WildflyPreviewNightly

So we can handle that separately.

Reply:
I have found the root cause of MicroProfileConfigConvertersTestCase issue, the custom module relies on javax.annotation that needs transformation. I am looking into it.

Reply:
The exclusions are now removed, the fix also applies to ts.ee9 profile.